.teamcity/_self/projects/WebApp.kt

```diff
@@ -249,6 +249,7 @@ object RunAllUnitTests : BuildType({
 			name = "Run type checks"
 			executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
 			scriptContent = """
+				set -x
```

Comment:
I've noticed you adding `set -x` to a bunch of these -- do you think we should add it to `bashNodeScript` so that it's on by default?

Reply:
That's a good idea! 👍🏼